services-turf/src/test/java/com/mapbox/turf/TurfMeasurementTest.java

```diff
@@ -44,6 +48,21 @@
   private static final String TURF_ENVELOPE_FEATURE_COLLECTION = "turf-envelope/feature-collection.geojson";
   private static final String LINE_DISTANCE_MULTILINESTRING
     = "turf-line-distance/multilinestring.geojson";
+  private static final String TURF_CENTER_FEATURE_COLLECTION_IN = "turf-center/in/feature-collection.geojson";
+  private static final String TURF_CENTER_IMBALANCED_POLYGON_IN = "turf-center/in/imbalanced-polygon.geojson";
+  private static final String TURF_CENTER_LINESTRING_IN = "turf-center/in/linestring.geojson";
+  private static final String TURF_CENTER_POINT_IN = "turf-center/in/point.geojson";
+  private static final String TURF_CENTER_POINTS_WITH_WEIGHTS_IN = "turf-center/in/points-with-weights.geojson";
+  private static final String TURF_CENTER_POLYGON_IN = "turf-center/in/polygon.geojson";
+  private static final String TURF_CENTER_POLYGON_WITHOUT_WEIGHTS_IN = "turf-center/in/polygon-without-weights.geojson";
```

Comment:
very small nit: only list the `geojson` files that you're using in the tests below?

<img width="876" alt="Screen Shot 2019-05-17 at 2 52 38 PM" src="https://user-images.githubusercontent.com/4394910/57958094-76419780-78b3-11e9-8527-21c049b148d5.png">

Reply:
Nevermind. Forgot your mention of `need to fully implement the test suite` in OP